### PR TITLE
隐藏可视化编辑器标题

### PR DESCRIPTION
**Pull request by ZoruaFox**

### CODE REVIEW

**Comment by Blossomstripe** [comment]
I have a question, why does this pr require two reviewers, and I cannot bypass the protection?

**Comment by ZoruaFox** [comment]
because&nbsp;i&nbsp;set&nbsp;it&nbsp;to


Zorua
***@***.***

中国民用航空飞行学院





&nbsp;




------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "TopRealm/InterfaceCodes"                                                                                    ***@***.***&gt;;
Date:&nbsp;Thu, May 25, 2023 04:12 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [TopRealm/InterfaceCodes] 隐藏可视化编辑器标题 (PR #35)





 
I have a question, why does this pr require two reviewers, and I cannot bypass the protection?

File: src/others/Citizen.css [hunk 1]
```diff
@@ -101,5 +101,17 @@ toc:hover,
...
 }
 
+/* 可视化编辑器界面，隐藏标题 */
+.ve-activated h1#firstHeading {
+	display: none;
+}
+
+.ve-activated .citizen-body-header--sticky .mw-body-header::before {
+	display: none;
+}
```
Comment (Blossomstripe) [inline src/others/Citizen.css:111]: 能否介绍一下这三行的作用？

Comment (ZoruaFox) [reply src/others/Citizen.css:111]: VKLK$9F13D`F5_66{EEJ~VE

**Review by Blossomstripe — APPROVED** [review]

File: src/others/Citizen.css (hunk 1 above)
Comment (Blossomstripe) [reply src/others/Citizen.css:111]: “手机端滚动的时候避免置顶标题与编辑工具栏冲突的，否则极少数情况下这个白边会和编辑工具栏冲突”
——兰江水@求闻百科

**Pull request merged**